__tests__/utils/ValidatorTest.js

```diff
@@ -0,0 +1,223 @@
+import Validator from '../../src/utils/Validator';
+import RACING_GAME from '../../src/constants/racingGame';
+import CustomError from '../../src/errors/error';
+
+describe('Validator 테스트', () => {
+  describe('is로 시작하는 메서드는 boolean을 반환한다.', () => {
+    describe('isEmptyString()', () => {
+      // given
+      const cases = [
+        {
+          input: '',
+          expected: true,
+        },
+        {
+          input: 'not empty string :)',
+          expected: false,
+        },
+      ];
+
+      cases.forEach(({ input, expected }) => {
+        test(`빈 문자열인지 확인한다.`, () => {
+          // when
+          const result = Validator.isEmptyString(input);
+
+          // then
+          expect(result).toBe(expected);
+        });
+      });
+    });
+
+    describe('isValidCarNameLength()', () => {
+      // 자동차의 이름은 1자 이상 5자 이하만 가능하다.
+
+      // given
+      const cases = [
+        {
+          input: 'pobi',
+          expected: true,
+          description: '정상적으로 생성되는 경우',
+        },
+        {
+          input: 'crong',
+          expected: true,
+          description: '정상적으로 생성되는 경우',
+        },
+        {
+          input: '',
+          expected: false,
+          description: '길이가 0인 경우',
+        },
+        {
+          input: 'pengoose',
+          expected: false,
+          description: '길이가 5를 초과하는 경우',
+        },
+      ];
+
+      const isValidLength = (name) => Validator.isValidCarNameLength(name);
+
+      cases.forEach(({ input, expected, description }) => {
+        test(`${description}, ${expected}를 반환한다.`, () => {
+          // when
+          const result = isValidLength(input);
+
+          // then
+          expect(result).toBe(expected);
+        });
+      });
+    });
+  });
+
+  describe('validate로 시작하는 메서드는 CustomError를 생성한다.', () => {
+    /**
+     *
+     * @param {() => void} validation 유효성 검증 함수입니다.
+     * @param {*} input 검증할 값을 입력합니다.
+     * @param {CustomError} error 생성되어야하는 에러입니다.
+     */
+    const shouldThrowError = (validation, input, error = CustomError) => {
+      // when
+      const validate = () => validation(input);
+
+      // then
+      expect(validate).toThrow(error);
```

Comment:
`validation(input)`이 복잡하지 않기 때문에 `validate`라는 변수에 담지 않고 바로 화살표 함수를 써보는 건 어떨까요?
```suggestion
      // then
      expect(() => validation(input)).toThrow(error);
```

Reply:
저번주에 다른 분들 코드를보니 given, when, then을 잘 쪼개두셨길래...
조금 따라해보고 싶었습니다 머쓱 ㅋㅋㅋㅋ

생각없이 너무 분리만 하려고 했더니 여기저기서 허점들이 보이는군요 후후.
차주에 참고하겠습니다 :)

```tsx
 // when & then
 expect(() => validation(input)).toThrow(error);
```

😄👍